alicat/util.py

```diff
@@ -177,7 +165,7 @@ class SerialClient(Client):
     def __init__(self,
                  address: str,
                  baudrate: int = 19200,
-                 timeout: float = 0.15,
+                 timeout: float = 10.0,
```

Comment:
Actually using `self.timeout` instead of hard-coded values is an improvement over the existing hard-coded values, and I have no reason the old code wasn't.

However, 10.0s is _way_ too long here for a serial device.  Even the ancient firmware versions respond much faster than that, and fairly deterministically.

Reply:
I totally agree, but sometimes/rarely/occasionally my raspberry pi seems to need quite a long time to connect to the device; particularly after connecting/disconnecting from a different device.  For example, with a timeout of 0.15, `test_driver_cli` fails when run consecutively on two of my devices.  The first connects and responds immediately, but then when the second device is tested immediately after, I get a FAIL.  One and exactly one time I had to wait upwards of 5 seconds and the second device successfully connected.

This timeout is buried deep within the module and not accessible to users, so I figured way overdoing it was better than setting a short timeout that seems reasonable but for some reason _sometimes_ fails on real hardware.

Reply:
https://github.com/alexrudd2/alicat/blob/master/alicat/driver.py#L47

The intention here (although I haven't tested it recently) is that you can do `FlowController(address='xxx', timeout=xxx)` and the timeout value will be passed via `**kwargs` all the way down to the client.

Reply:
>  particularly after connecting/disconnecting from a different device

This may well be a separate bug!